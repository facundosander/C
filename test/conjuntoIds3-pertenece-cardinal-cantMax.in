# crea un conjunto con rango 1..10
crearConjunto 10

# verifica cantidad m√°xima del conjunto (10)
cantMaxConjunto

# inserta 7 elementos
insertarConjunto 1
insertarConjunto 2
insertarConjunto 3
insertarConjunto 5
insertarConjunto 6
insertarConjunto 7
insertarConjunto 9
# imprime el conjunto
imprimirConjunto 

# verifica dos elementos que si pertenecen
perteneceConjunto 3
perteneceConjunto 7

# verifica dos elementos que no pertenecen y estan en el rango
perteneceConjunto 4
perteneceConjunto 8

# verifica dos elementos que no pertenecen y no estan en el rango
perteneceConjunto 0
perteneceConjunto 11

# imprime el cardinal del conjunto 
cardinalConjunto

# borra dos elementos e imprime el cardinal
borrarConjunto 1
borrarConjunto 5
cardinalConjunto

# verifica que los elementos borrados no esten mas 
perteneceConjunto 1
perteneceConjunto 5

# libera el conjunto 
liberarConjunto 
Fin
