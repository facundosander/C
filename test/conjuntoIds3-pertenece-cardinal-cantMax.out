1>#  crea un conjunto con rango 1..10.
2>3>#  verifica cantidad máxima del conjunto (10).
4>La cantidad máxima del conjunto es: 10.
5>#  inserta 7 elementos.
6>7>8>9>10>11>12>13>#  imprime el conjunto.
14>1 2 3 5 6 7 9 
15>#  verifica dos elementos que si pertenecen.
16>El elemento 3 pertenece al conjunto.
17>El elemento 7 pertenece al conjunto.
18>#  verifica dos elementos que no pertenecen y estan en el rango.
19>El elemento 4 no pertenece al conjunto.
20>El elemento 8 no pertenece al conjunto.
21>#  verifica dos elementos que no pertenecen y no estan en el rango.
22>El elemento 0 no pertenece al conjunto.
23>El elemento 11 no pertenece al conjunto.
24>#  imprime el cardinal del conjunto .
25>El cardinal del conjunto es: 7.
26>#  borra dos elementos e imprime el cardinal.
27>28>29>El cardinal del conjunto es: 5.
30>#  verifica que los elementos borrados no esten mas .
31>El elemento 1 no pertenece al conjunto.
32>El elemento 5 no pertenece al conjunto.
33>#  libera el conjunto .
34>35>Fin.
